tslint.json

```diff
@@ -21,10 +21,6 @@
       "avoid-escape",
       "jsx-double"
     ],
-    "no-unused-variable": [
```

Comment:
It is deprecated in TSLint. You're prompted to start using the compilerOption --noUnusedParameters and the like.